template/preact.config.js

```diff
@@ -11,17 +11,14 @@ export default {
      * @param {object} options - this is mainly relevant for plugins (will always be empty in the config), default to an empty object
      **/
     webpack(config, env, helpers, options) {
-        // Switch css-loader for typings-for-css-modules-loader, which is a wrapper
-        // that automatically generates .d.ts files for loaded CSS
-        helpers.getLoadersByName(config, "css-loader").forEach(({ loader }) => {
-            loader.loader = "typings-for-css-modules-loader";
-            loader.options = Object.assign(loader.options, {
-                camelCase: true,
+        config.module.rules[4].use.splice(1, 0, {
+            loader: "@teamsupercell/typings-for-css-modules-loader",
+            options: {
                 banner:
                     "// This file is automatically generated from your CSS. Any edits will be overwritten.",
-                namedExport: true,
-                silent: true
-            });
+                formatter: "prettier",
```

Comment:
Would prefer to leave this out. If prettier is installed, it will already default to using Prettier.

```suggestion
```